index.js

```diff
@@ -88,16 +91,26 @@ class Geocoder extends Component {
         })
       }
     } else {
-      search({
+      const autocompleteQuery = {
         apiKey,
         boundary,
         focusPoint,
         text: input
-      }).then((geojson) => {
+      }
+      const autocompleteQueryKey = stableStringify(autocompleteQuery)
```

Comment:
I know that but it's a contrived example. I was wondering if you actually found it to be unstable in practice.